CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   use new public method `generateManifest()`.
 - Fixed an issue where an immediate `<style>` child of `<dom-module>` was
   not moved into generated `<template>`.
+- BREAKING: Added option `rewriteTemplateUrls` to support old-style rewriting
```

Comment:
Good idea.  Done.  I also changed the name now to `rewriteUrlsInTemplates` which I think is more intention-revealing.